.github/workflows/tests.yml

```diff
@@ -24,36 +24,15 @@ jobs:
       fail-fast: false  # if one fails, keep the other(s) running
     name: "🛃 Unit tests (Fedora ${{ matrix.fedora_version }})"
     runs-on: ubuntu-latest
-    container:
```

Comment:
~This change is not explained in the commit message AFAICT, why is this removed?~ Now that I read the entire commit I see what is happening. I personally would expand the commit message a bit and describe that this commit replaces the GH action container/run with a makefile target that also builds a container/runs things insides the container.

Reply:
… enriched the commit message